webrtc/videoroom/mix.exs

```diff
@@ -20,14 +21,28 @@ defmodule VideoRoom.MixProject do
 
   defp deps do
     [
+      {:membrane_core, github: "membraneframework/membrane_core", override: true},
       {:membrane_rtc_engine, github: "membraneframework/membrane_rtc_engine"},
+      {:esbuild, "~> 0.1", runtime: Mix.env() == :dev},
       {:plug_cowboy, "~> 2.0"},
-      {:phoenix, "~> 1.5"},
-      {:phoenix_html, "~> 2.14"},
+      {:phoenix, "~> 1.6"},
+      {:phoenix_html, "~> 3.0"},
+      {:phoenix_live_view, "~> 0.16.0"},
       {:phoenix_live_reload, "~> 1.2"},
       {:jason, "~> 1.2"},
       {:phoenix_inline_svg, "~> 1.4"},
       {:uuid, "~> 1.1"}
     ]
   end
+
+  defp aliases do
+    [
+      setup: ["deps.get", "cmd --cd assets npm install"],
```

Comment:
this modifies package_lock, see how rtc engine does this

Reply:
Changed to `npm ci` if updating package lock is a problem